main/acle.md

```diff
@@ -2265,6 +2268,15 @@ is implementation-defined.
 The generic itanium C++ ABI, which we use in AArch64, already handles
 arrays with arbitrarily aligned elements
 
+## Scalable Vector Extension procedure call standard
+
+On SVE enabled AArch64 targets, the AAVPCS allows for procedure calls to
```

Comment:
AAVPCS is the Vector PCS of the Vector Function ABI. Did you mean AAPCS?

Reply:
Changed to AAPCS